eq-author-api/src/validation/createValidationError.js

```diff
@@ -113,6 +113,10 @@ module.exports = (dataPath, field, errorCode, questionnaire) => {
           rule = rules[dataPath[index + 2]];
           validationErr.routingRuleId = rule.id;
           validationErr.type = "routing";
+
+          if (rule.expressionGroup) {
+            validationErr.expressionGroupId = rule.expressionGroup.id;
+          }
```

Comment:
Does this fix that caching issue we were having?

Reply:
No, that's just to help make it easier to catch expression group errors, 'cause some of the AC for this ticket is taking multiple rules in a group into account.